### DISCUSSION TITLE
Tasmota ws2812 Multibe output Pin

### DISCUSSION THREAD

**Discussion by Smoeller0** [arendst/Tasmota#16427] [Q&A] [answered]
Hello
I need 4 Rgb led's on Tasmota. I Chose ws2812. But Every ws2812 led have connection to the esp32. Also every Led D In goes to a Pin on the esp32. 
In The Tasmota Config i have 4 times Ws2812 on the Output pin.
When i set every ws2812 on /1 every led work but all led do the same color.
When i set ws2812 1 - 4 i only can control the led 1 
How is it possibe to controll the leds on diffrent wires?

**Comment by sfromis** [comment, marked as answer]
Base Tasmota only supports one WS2812 pin, where you can have multiple addressable pixels. With ESP32, you also have the Berry language, including support for WS2812 on multiple pins, with control via Berry code instead of the "classic" commands.
